### DISCUSSION TITLE
As an Administrator I want to use the Central Server and Security Server UIs in Russian so that I can use my preferred language.

### DISCUSSION THREAD

**Discussion by Bibigozel** [nordic-institute/X-Road#2490] [Ideas]
as X-Road is used in several countries where Russian is a common language many administrators would find it more convenient to use the X-Road components in their preferred language.
We are prepared to submit the relevant pull request to fulfill this enhancement and provide later updates on X-Road new releases.

**Comment by petkivim** [comment]
Thank you for the heads up regarding your upcoming pull request. We appreciate your contribution.

In order to ensure a smooth contribution process, please review the Contributor License Agreement (CLA) and initiate the CLA signing process by following the instructions available here:

https://github.com/nordic-institute/X-Road/blob/develop/CONTRIBUTING.md#sign-the-cla

When opening the pull request on GitHub, please use issue ID `XRDDEV-2810` as a reference. It's a placeholder issue for your contribution:

https://nordic-institute.atlassian.net/browse/XRDDEV-2810

Detailed instructions for adding support for a new UI language are available here:

https://nordic-institute.atlassian.net/wiki/spaces/XRDKB/pages/864288788/How+to+Add+a+New+Language+to+the+Central+Server+or+Security+Server+UI
